packages/http/src/validator/__tests__/index.spec.ts

```diff
@@ -178,6 +186,46 @@ describe('HttpValidator', () => {
         it("validates query assuming it's empty", test(undefined, { url: { path: '/' } }));
       });
     });
+
+    describe('invalid schema provided', () => {
+      it('should not call the validate method', async () => {
```

Comment:
`should not call validateAgainstSchema method` ?
because we expect `.validate` to be called

Reply:
That is correct, I didn't pick a great selection of names.

Refactored in 653221e